Meter.c

```diff
@@ -96,46 +96,44 @@ static void BarMeterMode_draw(Meter* this, int x, int y, int w) {
    // The text in the bar is right aligned;
    // Pad with maximal spaces and then calculate needed starting position offset
    RichString_begin(bar);
-   RichString_appendChr(&bar, 0, ' ', w);
+   RichString_appendChr(&bar, 0, ' ', (unsigned int)w);
```

Comment:
This looks overall kinda noisy. The source code shouldn't be a casting show … ;-)

Reply:
That's the problem of policy and whether the compiler would complain about implicit casts to unsigned. Given there is a conditional `if (w < 1) return;` a few lines earlier, compiler should detect that the `w` value cannot go negative in this case, but I can't say for sure.

If htop is compiled with gcc or clang's `-Wsign-conversion` flag, will that cause a warning?